src/metabase/lib/drill_thru/underlying_records.cljc

```diff
@@ -163,15 +162,6 @@
                 (:sum-where :count-where :share)
                 [(last aggregation)]
 
-                ;; Metrics are standard filter + aggregation units; if the column is a metric get its filters.
-                ;; TODO metric v2: define behavior
-                #_#_:metric
-                  (-> (lib.metadata/legacy-metric query (last aggregation))
-                      :definition
-                      lib.convert/js-legacy-inner-query->pMBQL
-                      (assoc :database (:database query))
-                      (lib.filter/filters -1))
-
```

Comment:
I've deleted this because v2 metrics bring their filters int the source, not attached to the metric reference.